src/call_frame.rs

```diff
@@ -19,28 +19,62 @@ pub struct CallFrame {
     pub aux_heap: Heap,
     pub code_page: Vec<U256>,
     pub pc: u64,
-    /// Storage for the frame using a type that implements the Storage trait.
-    /// The supported types are InMemory and RocksDB storage.
+    // TODO: Storage is more complicated than this. We probably want to abstract it into a trait
+    // to support in-memory vs on-disk storage, etc.
     pub storage: Rc<RefCell<dyn Storage>>,
-    /// Transient storage should be used for temporary storage within a transaction and then discarded.
-    pub transient_storage: InMemory,
     pub gas_left: Saturating<u32>,
+    pub transient_storage: InMemory,
```

Comment:
Same here, there are some comments missing

Reply:
I'll add it back!